test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex route
```

Comment:
I think this reads really awkwardly and requires us to pass information which from a test perspective should not be needed. Let's back off a bit and look like how this should/could read:

    Scenario: User can access application via HTTP
      Given Minishift has state "Running"
        And "ruby-ex" exposes route "foo"
       Then status code of HTTP request to "/" of "foo" is equal to "200"
        And body of HTTP request to "/" of "foo" contains "Welcome to your   Ruby application on OpenShift"

This is imo concise. It also takes hight for the fact, that one might want to hit something else than the root context "/". 

What's about trying to aim for something like that?